src/main/java/com/denizenscript/depenizen/bukkit/events/breweryx/BreweryChatDistortScriptEvent.java

```diff
@@ -0,0 +1,81 @@
+package com.denizenscript.depenizen.bukkit.events.breweryx;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.depenizen.bukkit.objects.breweryx.BPlayerTag;
+import com.dre.brewery.api.events.PlayerChatDistortEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class BreweryChatDistortScriptEvent extends BukkitScriptEvent implements Listener {
+    public BreweryChatDistortScriptEvent() {
+    }
+
+    // <--[event]
+    // @Events
+    // brewery chat distort
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a drunk brewery player's chat is distorted.
+    //
+    // @Context
+    // <context.message> Returns the message to be sent after being distorted.
+    // <context.original_message> Returns the original message before being distorted.
+    // <context.bplayer> Returns an BPlayerTag of the player that drank the brew.
+    // <context.player> Returns a PlayerTag of the player that had their chat distorted.
+    //
+    // @Determine
+    // ElementTag(String) to set the message to be sent after being distorted.
+    //
+    // @Plugin Depenizen, BreweryX
+    //
+    // @Player Always.
+    //
+    // @Group Depenizen
+    //
+    // -->
+    public PlayerChatDistortEvent event;
+    public PlayerTag playerTag;
+
+    @Override
+    public boolean couldMatch(ScriptPath path) {
+        return path.eventLower.startsWith("brewery chat distort");
```

Comment:
this is the legacy scriptevent format, please check the code in Denizen-Core, Denizen, or dDiscordBot for modern examples